torch/_dynamo/testing.py

```diff
@@ -215,7 +221,7 @@ def standard_test(self, fn, nargs, expected_ops=None, expected_ops_dynamic=None)
     self.assertTrue(same(val1b, correct1))
     self.assertTrue(same(val2a, correct2))
     self.assertTrue(same(val2b, correct2))
-    self.assertEqual(actual.frame_count, 1)
+    self.assertEqual(actual.frame_count, expected_frame_count)
```

Comment:
No, let's keep this at 1 - this test explicitly uses `optimize_assert` for that end - if you want frame counts of > 1, check out usage of compile counter.

I would be open to a version of this fn that calls `torch._dynamo.optimize(compile_counter)` - search for "torch._dynamo.testing.CompileCounter()" in the suite for examples.

Reply:
Ah I see - `optimize_assert` ensures no graph breaks. I reverted this change and tweaked my tests slightly to have a single operation instead of being constant folded out when dynamic shapes are disabled.